test/export/dashboard.test.ts

```diff
@@ -0,0 +1,159 @@
+// (C) 2019 GoodData Corporation
+import "isomorphic-fetch";
+import fetchMock from "fetch-mock";
+import { DashboardModule } from "../../src/export/dashboard";
+import { ApiResponseError, XhrModule } from "../../src/xhr";
+import {
+    ACCEPTED_REQUEST_STATUS,
+    BAD_REQUEST_STATUS,
+    SUCCESS_REQUEST_STATUS,
+} from "../../src/constants/errors";
+import { mockPollingRequest, mockPollingRequestWithStatus } from "../helpers/polling";
+import { IExportResponse } from "../../src/interfaces";
+import { DashboardExport } from "@gooddata/typings";
+
+const dashboardModuleMock = () => new DashboardModule(new XhrModule(fetch, {}));
+
+describe("exportDashboard", () => {
+    const projectId = "testProjectId";
+    const dashboardUri = `/gdc/md/${projectId}/obj/dashboard123`;
+    const endpoint = `/gdc/internal/projects/${projectId}/exportDashboard`;
+    const createdPdf = `/gdc/exporter/result/${projectId}/pdfFileUri`;
+
+    const relativeDateFilter: DashboardExport.FilterContextItem = {
+        dateFilter: {
+            type: "relative",
+            from: -11,
+            to: 11,
+            granularity: "GDC.time.month",
```

Comment:
Please create constant for `GDC.time.month` and `relative`, `absolute`, or re-use if they are existed

Reply:
Could you please explain why?
I think this is raw mock data, so we can define it directly.
And in case we define new, should it be only in this local test file? I don't use it anywhere else in my impl though.

Reply:
to prevent typo mistake and easier to maintain later .
The current texts are ok to me